.gitignore

```diff
@@ -1,6 +1,7 @@
+.build
 yet-another-cloudwatch-exporter
 !charts/yet-another-cloudwatch-exporter
 vendor
 dist
 /yace
-.idea
+*.tar.gz
```

Comment:
We could still continue to gitignore the `.idea` folder I guess?

Reply:
We have a community policy to not include these editor patterns in our configuration. There are just too many variations to support.

This is also better handled by the developer's git configuration. We have a "canned response" for theses requests.

```
Please don't include your local editor ignores in the repo. Use [`git config --global core.excludesFile`](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).
```